src/test/groovy/org/osiam/resources/exceptions/OsiamExceptionHandlerSpec.groovy

```diff
@@ -26,89 +26,69 @@ package org.osiam.resources.exceptions
 import com.fasterxml.jackson.databind.ObjectMapper
 import org.osiam.resources.scim.User
 import org.springframework.http.HttpStatus
-import org.springframework.web.context.request.WebRequest
 import spock.lang.Specification
 
 class OsiamExceptionHandlerSpec extends Specification {
```

Comment:
Please add tests for `handleInvalidConstraintException`, `handleResourceExists` and `handleBackendFailure`